src/components/Buttons.js

```diff
@@ -58,6 +61,7 @@ const Buttons = () => {
           <button
             type="button"
             className="btn number"
+            data-testid="btn-7"
```

Comment:
- [ ] Please it will nice to update your `readme file` with steps to clone and run the tests locally.